src/main/java/com/epam/preprod/sirenko/DAO/ProductsInDBDAO.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.preprod.sirenko.DAO;
```

Comment:
Please rename DAO directory to dao (with lovercase) or repository

Reply:
done